### PR TITLE
Create Nullable_Decimal_Issue.cs

### PR DESCRIPTION
**Pull request by emiliovmq**

### CODE REVIEW

File: test/FastExpressionCompiler.IssueTests/Nullable_Decimal_Issue.cs [hunk 1]
```diff
@@ -0,0 +1,30 @@
+using Xunit;
+using using System.Linq.Expressions;
```
Comment (dadhi) [inline test/FastExpressionCompiler.IssueTests/Nullable_Decimal_Issue.cs]: `using using`?

Comment (emiliovmq) [reply test/FastExpressionCompiler.IssueTests/Nullable_Decimal_Issue.cs]: Ups...I'm sorry...I just copied and pasted and didn't realized that error. I'm sorry, I'll fix it in a moment..

**Comment by dadhi** [comment]
Hmm, this is maybe a bug in LightExpression that `Expression.Equal` returns `Expression` instead of `BinaryExpression`.

But for now, in order for your PR to compile, could you replace explicits types with `var`s?

**Comment by emiliovmq** [comment]
Sure...

**Pull request merged**